### DISCUSSION TITLE
Is it possible to view multiple PDF files as one "endless" PDF file

### DISCUSSION THREAD

**Discussion by Peter-Franz** [mozilla/pdf.js#18431] [Q&A]
At first thank you all for your work the pdf.js is the best pdf viewer for my work. 

I often had to open different PDF files in one folder. Mostly that are PDFs with one Page in the same Size, so I have to open the files in different windows or tabs with is not the best way to work on one file to another. 

My workaround is to use a tool to merge this files and then open the merge PDF. But I want to ask if there is a way to speed this process.

So, is there a way to show all PDFs of a Folder as one "endless" PDF file?

**Comment by catmanjan** [comment]
Also interested in this possibility - anyone managed to get it working?
